tests/Fixtures/Integration/misc/phpdoc_to_param_type.test

```diff
@@ -0,0 +1,17 @@
+--TEST--
+Integration of fixers: phpdoc_to_param_type.
+--RULESET--
+{"phpdoc_to_param_type": true}
```

Comment:
The purpose of these integration tests is to check that several fixers work fine when used together. This test only uses `phpdoc_to_param_type` so it can be deleted (the scenario is already covered by unit tests of the fixer I think).